src/scala/org/pantsbuild/scoverage/report/ScoverageReport.scala

```diff
@@ -0,0 +1,254 @@
+package org.pantsbuild.scoverage.report
+
+import java.io.File
+import sbt.internal.util.ConsoleLogger
+import org.apache.commons.io.FileUtils
+
+import scoverage.{ Coverage, IOUtils, Serializer }
+import scoverage.report.{ ScoverageHtmlWriter, ScoverageXmlWriter }
+
+object ScoverageReport {
+  val Scoverage = "scoverage"
+
+  // Copying the logger from org.pantsbuild.zinc.bootstrap
+  // As per https://github.com/pantsbuild/pants/issues/6160, this is a workaround
+  // so we can run zinc without $PATH (as needed in remoting).
+  System.setProperty("sbt.log.format", "true")
+
+  // Setting the logger
+  val logger = ConsoleLogger.apply()
+
+  case class ReportOptions(
```

Comment:
This class looks redundant with the `Settings` class: would it be reasonable to delete it and add a `def validate` or some `lazy vals` to the Settings class if need be?

Reply:
Yup true, It wasn't necessary. Thanks